src/app/shared/components/paginator/paginator.component.ts

```diff
@@ -10,14 +12,21 @@ import { Pageable } from '@app/core/api';
 })
 export class PaginatorComponent {
   @Input() length: number | undefined;
-  @Output() param: EventEmitter<Pageable> = new EventEmitter<Pageable>();
+  @Output() param: EventEmitter<void> = new EventEmitter<void>();
 
-  constructor() {}
+  public params$: Observable<Pageable>;
 
-  handlePageEvent(event: PageEvent) {
-    this.param.emit({
+  constructor(private router: Router, private route: ActivatedRoute) {
+    this.params$ = this.route.queryParams;
+  }
+
+  async handlePageEvent(event: PageEvent) {
+    const paginator = {
       page: event.pageIndex,
       size: event.pageSize,
-    });
+    };
+    localStorage.setItem('size', JSON.stringify(event.pageSize));
```

Comment:
`size` is a very generic term and `localStorage` is global per domain. Quite often, lots of third party libraries are loaded which can also use `localStorage`. I think we should prefix all the fields, e.g. `pomagamukrainie-size`?